steward/src/commands/tx/cosmos.rs

```diff
@@ -25,9 +25,20 @@ impl Runnable for Cosmos {
     }
 }
 
+/// Send tx from Cosmos to Ethereum chain
 #[derive(Command, Debug, Parser)]
+#[clap(
+    long_about = "DESCRIPTION \n\n Send transactions from Cosmos to Ethereum chain.\n This command sends tx from the Cosmos chain to the Eth chain.\n It takes a free vector of strings as required flags for Cosmos key, Eth key and amount."
```

Comment:
Since we've converted to flags none of the `tx` commands take a free vector of strings any more.

Reply:
This bit still refers to the flags here as a "free vector of strings".